hw/top_earlgrey/cdc/cdc_waivers.spi_device.tcl

```diff
@@ -0,0 +1,25 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Verix CDC waiver file
+# Expression:
+#  ControlSignal==""
+#  ReconSignal==""
+#  MultiClockDomains=="IO_DIV2_CLK::IO_DIV4_CLK"
+
+set_rule_status -rule {W_RECON_GROUPS} -status {Waived}         \
+   -expression {(ControlSignal=~"*u_status_23_to_1_sync.*") &&       \
+                (ReconSignal=~"*u_spid_status.outclk_p2s_byte_*")} \
+   -comment {status bits are converged into SPI line. But transmit bit-by-bit. OK to send old and new status bits in a transaction}
+
+set_rule_status -rule {W_CNTL} -status {Waived}   \
```

Comment:
ah this is the part we talked about if there's a specific naming inside we can waive in general? 
Is there not a rule somewhere in verix where we can inform the tool what is a legal synchronizer?

Reply:
unfortunately, looks like no. But I should read the tool manual thoroughly. Maybe I missed a few commands.

Reply:
`set_user_cntl_synchronizer` may be this is what you expect. I will take a look

Reply:
yeah but we can merge this first.

Reply:
Looks like this is addressed in #11655? 
Does it make sense to merge the common waivers first and then land this without the `W_CNTL` waivers for paths through the synchronizer?

Reply:
The PR #11655 does not work at this time. I am figuring out how nicely I can specify uniquified synchronizer modules.

Reply:
Oh I see. Nevermind then.